pyproject.toml

```diff
@@ -125,6 +125,7 @@ watchdog = ">=0.9.0"
 pyshacl = { version = "^0.25.0", optional = true }
 black = { version=">=24.0.0", optional = true }
 typing-extensions = { version=">=4.4.0", python="<3.9" }
+testcontainers = "3.7.1"
```

Comment:
I think this should be a test dependency only?

Reply:
Indeed, I'm going to update that right now 👍